cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -100,6 +100,9 @@ pub enum Inst {
         loc: SourceLoc,
     },
 
+    /// Do a sign-extend based on the sign of the value in al into ah: (cbw)
+    SignExtendAlAh,
```

Comment:
We should reuse the `SignExtendRaxRdx` instruction, because this new instruction you added and the former have the same semantics: sign-extend the "data" (originally on x86, `rax` is the "data" (Accumulator, hence A) register up to 32 bits, and when more is required, the `rdx` contains the rest of the Data (hence D in name)). Maybe rename `SignExtendRaxRdx` to `SignExtendData`, and make the size an `Option` to indicate al->ah? (or use 0 for al->ah sign-extension?) Or we could have a `SignExtendOpSize` enum that contains all the valid sizes for this instruction?